src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/timeline/BookingTimelineService.kt

```diff
@@ -209,6 +224,153 @@ class BookingTimelineService(
       .groupBy { it.date }
   }
 
+  private fun applyTrancheAdjustmentLogic(
```

Comment:
Could this entire functionality be lifted into a new file?

Reply:
yes I think that would make sense